todo_app/test_tasks.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import todo_app.view_model as view_model
+import datetime
+import todo_app.Task as task
+    
+@pytest.fixture
+def test_tasks():
+    today = datetime.date.today()
+    yesterday = today - datetime.timedelta(days=1)
+    task_list = [
+        task.Task(1, 'To Do', 'Task1', str(today)),
+        task.Task(2, 'To Do', 'Task2', str(today)),
+        task.Task(3, 'Doing', 'Task3', str(today)),
+        task.Task(4, 'Doing', 'Task4', str(today)),
+        task.Task(5, 'Doing', 'Task5', str(today)),
+        task.Task(6, 'Done', 'Task6', str(today)),
+        task.Task(7, 'Done', 'Task7', str(today)),
+        task.Task(8, 'Done', 'Task8', str(today)),
+        task.Task(9, 'Done', 'Task8', str(today)),
+        task.Task(10, 'Done', 'Task10', str(yesterday)),
+        task.Task(11, 'Done', 'Task11', str(yesterday)),
+        task.Task(12, 'Done', 'Task12', str(yesterday))
+    ]
+
+    test_list = view_model.ViewModel(task_list)
+
+    return test_list
+
+@pytest.fixture
+def test_tasks_2():
+    today = datetime.date.today()
+    yesterday = today - datetime.timedelta(days= 1)
+    task_list = [ 
+        task.Task(1, 'To Do', 'Task1', str(today)),
+        task.Task(2, 'To Do', 'Task2', str(today)),
+        task.Task(3, 'Doing', 'Task3', str(today)),
+        task.Task(4, 'Doing', 'Task4', str(today)),
+        task.Task(5, 'Done', 'Task5', str(today)),
+        task.Task(6, 'Done', 'Task6', str(yesterday)),
+        task.Task(7, 'Done', 'Task7', str(yesterday)),
+        task.Task(8, 'Done', 'Task8', str(yesterday))
+    ]
+    
+    test_list = view_model.ViewModel(task_list)
+
+    return test_list
+
+@pytest.fixture
+def test_tasks_3():
```

Comment:
Clean code comment; the names of these don't particularly reflect their purpose which seems to be varying numbers of 'Done' tasks. These might be clearer as e.g.
`test_tasks_with_5_done` etc.